File: Client/ies/src/app/models/test-result.ts
export class TestResult {
    studentId: number
    date: Date
    wrongQuestionIds: number[]
    score: number
    startingChapterId: number
    endingChapterId: number

    constructor() {
        this.studentId = 0;
        this.date = new Date();
        this.wrongQuestionIds = []
        this.score = 0
        this.startingChapterId = 0
        this.endingChapterId = 0
    }
}

File: Server/InteractiveEducationalSoftware/InteractiveEducationalSoftware/IES.Models/BusinessModels/TestResult.cs
namespace IES.Models.BusinessModels
{
    public class TestResult
    {
        public int StudentId { get; set; }
        public DateTime Date { get; set; }
        public List<int> WrongQuestionIds { get; set; }
        public int Score { get; set; }
        public int StartingChapterId { get; set; }
        public int EndingChapterId { get; set; }
    }
}
